lib/ExpensiMark.js

```diff
@@ -73,10 +75,7 @@ export default class ExpensiMark {
             {
                 name: 'link',
                 process: (textToProcess, replacement) => {
-                    const regex = new RegExp(
-                        `\\[([^\\][]*(?:\\[[^\\][]*][^\\][]*)*)]\\(${MARKDOWN_URL_REGEX}\\)(?![^<]*(<\\/pre>|<\\/code>))`,
-                        'gi',
-                    );
+                    const regex = new RegExp(MARKDOWN_LINK_REGEX, 'gi');
                     return this.modifyTextForUrlLinks(regex, textToProcess, replacement);
```

Comment:
To clarify, here we can just pass `MARKDOWN_LINK_REGEXP` as the first param

Reply:
like this?
```
    return this.modifyTextForUrlLinks(MARKDOWN_LINK_REGEX, textToProcess, replacement);
```

Reply:
Done